apps/cow-fi/pages/learn/[article].tsx

```diff
@@ -119,6 +116,29 @@ function getRandomArticles(articles: Article[], count: number): Article[] {
   return shuffled.slice(0, count)
 }
 
+export function ArticleSharedRichTextComponent({ sharedRichText }: { sharedRichText: SharedRichTextComponent }) {
+  const { replaceImageUrls, LazyImage } = useLazyLoadImages()
+
+  // Preprocess the HTML content
+  const processedContent = React.useMemo(() => {
```

Comment:
Just to be consistent with the rest of code-base, please use it as `import { useMemo } from 'react'`